masterbase/guards.py

```diff
@@ -84,4 +85,7 @@ async def valid_session_guard(connection: ASGIConnection, _: BaseRouteHandler) -
     try:
         Server.query_from_params(api_key, converted_fake_ip, fake_port)
     except KeyError:
-        raise NotAuthorizedException("Cannot accept data from a non-existent gameserver!")
+        try:
+            a2s_server_query(converted_fake_ip, fake_port)
+        except SourceError:
+            raise NotAuthorizedException("Cannot accept data from a non-existent gameserver!")
```

Comment:
Sweet, any reason we can't just nuke most of the code here in favor of the `SourceServer` object? Always fun to delete code

Reply:
The A2S querying doesn't use the Steam APIs and isn't inherent to SDR, so I wanted to leave both there. But if we can A2S query all servers including casual/MM servers and it proves just as good, then maybe we should just move to it?

Reply:
done 👍